homeassistant/components/deconz/climate.py

```diff
@@ -68,6 +69,8 @@
 
 DECONZ_TO_PRESET_MODE = {value: key for key, value in PRESET_MODE_TO_DECONZ.items()}
 
+DEFAULT_MIN_TEMP = 5
+DEFAULT_MAX_TEMP = 30
 
```

Comment:
It doesn't look like you have the dev environment set up correctly, else the commit hooks would have caught this.